packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -5862,17 +5858,17 @@ The metadata for a CSR
 | Property               | Description                                                  | DataType                                                                    | Nullable | Unique | Readonly | MinLength | MaxLength | Validation |
 | ----------------       | ------------------------------------------------------------ | --------------------------------------------------------------------------- | -------- | ------ | -------- | --------- | --------- | ---------- |
 | commonName             | Common name of the subject                                   | String                                                                      | TRUE     | FALSE  | FALSE    |           |           |            |
```

Comment:
Why is this one capitalized and the other descriptions are not?

Reply:
missed one?